docs/client/apidocs.rst

```diff
@@ -0,0 +1,110 @@
+Code-based documentation
```

Comment:
I'd name this section `API Reference`, like http://flask.pocoo.org/docs/0.12/#api-reference